loops.js

```diff
@@ -87,5 +99,13 @@
 // etc.
 // ==========================================
 
-
-
+for (let i = 1; i < 101; i++) {
+    if (i % 3 === 0 && i % 5 === 0 ) {
```

Comment:
Super! Mijn enige persoonlijke idee is dat dit: if ((i % 3 === 0) && (i % 5 === 0)) makkelijker te lezen is